interface/patient_file/encounter/forms.php

```diff
@@ -20,6 +20,10 @@
     $groupId = $therapy_group;
 }
 $attendant_id = $attendant_type == 'pid' ? $pid : $therapy_group;
+if(!(acl_check("groups","glog",false, 'view') OR acl_check("groups","glog",false, 'write'))){
```

Comment:
much better to combine these:
``````
acl_check("groups","glog",false, array('view','write'))